src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Models/LayoutSettings.cs

```diff
@@ -18,7 +18,7 @@ public class LayoutSettings
         public Components Components { get; set; }
     }
 
-    /// <summary>
+     /// <summary>
```

Comment:
This does not seem right?

Reply:
Also, non of the doc added gives any value. Seems like a waste of good lines?

Reply:
Split in to different files. Documentation is requried